onboarding/guide-to-getting-started.md

```diff
@@ -0,0 +1,94 @@
+
+## Your first week
+
+_Goal: I can ssh into gravity (staging and production). I can PR to a repo in Artsy’s Github org. I know what the engineering teams are, what their main focuses are, and who leads them. I know my way around the Artsy site._
+
+### First week checklist
+
+#### Dev environment:
+
+- [ ] Get invited to the Artsy Github org
+  _Ask your mentor or another engineer for help, or log into github using the it@artsymail.com account in 1Password and add yourself._
+- [ ] Set up your general dev environment (download your favorite text editor, terminal application, etc.) _If you don't have a favorite, ask for advice but we'll probably tell you to install vscode._
+- [ ] Run the script in https://github.com/artsy/potential/blob/master/scripts/setup for initial setup.
+- [ ] Set up (some) of our most critical projects
+  - [ ] Gravity (#platform-humans)
+  - [ ] Force (#front-end)
+  - [ ] Metaphysics (#front-end)
+  - [ ] Volt (#product-sell)
+  - [ ] Reaction (#front-end)
+  - [ ] Emission (#front-end-ios)
+    _For each of these, follow the instructions in the README. If you encounter problems, it’s not your fault! Ask in the appropriate slack channels (always default to #dev if you don’t know) for help and PR clarifying changes._
+- [ ] Make a PR to an Artsy repo (code change or simple doc update)
+- [ ] Keep your eye out for ways to improve our documentation-- including in this repo!
+- [ ] Ask an engineer for help getting set up with an AWS account. Follow the instructions in https://github.com/artsy/hokusai for getting set up with hokusai.
+- [ ] Successfully open a gravity staging and production console
+
+#### Product team:
+
+- [ ] Get access to [JIRA](https://artsyproduct.atlassian.net/) and explore each of the product team’s sprints
+- [ ] Scan through the issues labeled [“good-first-issue”](https://artsyproduct.atlassian.net/issues/?filter=-4&jql=labels%20%3D%20good-first-issue%20order%20by%20created%20DESC)
+- [ ] Read through the [project list](https://github.com/artsy/potential/wiki/Project-List)
+
+#### Reading/Learning:
+
+- [ ] Take a tour of Artsy. Log into [staging.artsy.net](https://staging.artsy.net) with your admin account and click around without fear. Visit each of the top-nav pages. Bid in an auction. Make an inquiry. _Ask your mentor or others for help with any of these concepts. Have them show you CMS and some internal apps, like Ohm or Torque._
+- [ ] Learn how code goes from your local machine --> production. _Talk to your mentor. Have them explain gravity's deployment process. Read about our [staging environment](https://github.com/artsy/gravity/blob/master/doc/StagingEnvironment.md).
+- [ ] Reading: [Working with the API](https://github.com/artsy/potential/blob/master/platform/WorkingWithGravity.md)
+- [ ] Scan our list of preferred technologies. Dig in where necessary.
+  - [ ] Javascriptures
```

Comment:
#TODO I need to add links for these

Reply:
Leave that to me, for now, link to this doc: https://github.com/artsy/README/blob/master/practices/front-end.md#team-information